### ISSUE TITLE
Request Rules Version explanation

### ISSUE THREAD

**Issue by Weskernation** [foundryvtt/dnd5e#4363]
![image](https://github.com/user-attachments/assets/c4f78825-1cc8-42d1-b11c-006f6a8c4a66)
Can the "Various" changes be specified. when hovering over it or something while inside foundry.

**Comment by JPMeehan** [comment]
I agree a more thorough explanation in-client would be helpful, but at least so you know it's covered by #3982
